examples/bank_reserves/batch_run.py

```diff
@@ -190,23 +190,27 @@ def run_model(self):
     data = batch_run(
         BankReservesModel,
         br_params,
-        model_reporters={"Rich": get_num_rich_agents},
-        agent_reporters={"Wealth": "wealth"},
     )
     br_df = pd.DataFrame(data)
     br_df.to_csv("BankReservesModel_Data.csv")
 
     # The commented out code below is the equivalent code as above, but done
     # via the legacy BatchRunner class. This is a good example to look at if
     # you want to migrate your code to use `batch_run()` from `BatchRunner`.
+    # Things to note:
+    # - You have to set "reserve_percent" in br_params to `[5]`, because the
+    #   legacy BatchRunner doesn't auto-detect that it is single-valued.
+    # - The model reporters need to be explicitly specified in the legacy
+    #   BatchRunner
     """
-    br = BatchRunner(
+    from mesa.batchrunner import BatchRunnerMP
+    br = BatchRunnerMP(
```

Comment:
Should this be converted to batch_run since we are deprecating BatchRunnerMP?

Reply:
The code here is intentionally left behind, so that people can compare how it is done via `batch_run` and `BatchRunnerMP`.

Reply:
Ahhh right... that is what I get for rushing and also trying to work on the script.